runtime-modules/argo-bridge/src/tests/mod.rs

```diff
@@ -0,0 +1,370 @@
+#![cfg(test)]
+
+use core::convert::TryFrom;
+
+use frame_support::{assert_err, assert_ok};
+use sp_runtime::BoundedVec;
+
+use crate::{
+    account, joy,
+    tests::mock::{increase_block_number_by, AccountId, Balance, Balances, BlockNumber, Test},
+    BridgeConstraints, BridgeStatus, RemoteAccount, RemoteTransfer,
+};
+
+use self::mock::{
+    with_test_externalities, with_test_externalities_custom_mint_allowance, ArgoBridge,
+    RuntimeOrigin,
+};
+use crate::Error;
+
+mod mock;
+
+#[test]
+fn request_outbound_transfer_success() {
+    with_test_externalities(|| {
+        let fee = joy!(10);
+        let remote_chains = BoundedVec::try_from(vec![1u32]).unwrap();
+        let parameters = BridgeConstraints {
+            operator_account: Some(account!(1)),
+            pauser_accounts: Some(vec![account!(2), account!(3)]),
+            bridging_fee: Some(fee),
+            thawn_duration: None,
+            remote_chains: Some(remote_chains),
+        };
+        assert_ok!(ArgoBridge::update_bridge_constrains(
+            RuntimeOrigin::root(),
+            parameters
+        ));
+
+        Balances::set_balance(RuntimeOrigin::root(), account!(1), joy!(1020), joy!(0)).unwrap();
+
+        let remote_account = RemoteAccount {
+            account: [0; 32],
+            chain_id: 1,
+        };
+        let result = ArgoBridge::request_outbound_transfer(
+            RuntimeOrigin::signed(account!(1)),
+            remote_account,
+            joy!(1000),
+            joy!(fee),
+        );
+        assert_ok!(result);
```

Comment:
I'm not sure what kind of assertions we can do in our tests, but could this also assert:
1. Proper balance change in the caller account
2. Proper change of mint allowance?